src/main/java/mate/academy/service/AuthenticationServiceImpl.java

```diff
@@ -1,12 +1,17 @@
 package mate.academy.service;
 
+import mate.academy.Main;
 import mate.academy.exception.AuthenticationException;
 import mate.academy.model.User;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class AuthenticationServiceImpl implements AuthenticationService {
+    private static final Logger logger = LogManager.getLogger(Main.class);
+
     @Override
     public User login(String login, String password) throws AuthenticationException {
-        //TODO: add corresponding log message about method login was called
+        logger.debug("Method login was called. Params: login={}", login);
         User user = findByLogin(login);
         if (!user.getPassword().equals(password)) {
```

Comment:
please remove redundant comment below
// this user identifier should be set by DB. We will use dummy data for this example